# Define the task
uint32 task_id
string task_name
uint32 priority
---
# Define the result
bool success
string end_result
---
# Define a feedback message
uint8 percentage
string intermediate_result
